visualization/test/test-page/App.jsx

```diff
@@ -84,8 +84,11 @@ import { probit } from "../__mock-data/probit";
                         predictedY={data.predictedYs}
                         trueY={data.trueY}
                         supportedBinaryClassificationAccuracyKeys={["accuracy_score", "balanced_accuracy_score","precision_score", "recall_score"]}
+                        supportedBinaryClassificationParityKeys={["selection_rate", "selection_rate_ratio", "zero_one_loss", "recall_score"]}
                         supportedRegressionAccuracyKeys={["mean_absolute_error", "r2_score", "mean_squared_error", "root_mean_squared_error"]}
                         supportedProbabilityAccuracyKeys={["auc", "root_mean_squared_error", "balanced_root_mean_squared_error", "r2_score", "mean_squared_error", "mean_absolute_error"]}
+                        supportedRegressionParityKeys={["selection_rate", "selection_rate_ratio", "zero_one_loss", "recall_score"]}
```

Comment:
I'm guessig this is just a first pass, but the disparity metrics here I think only apply to binary classifier, we'll need other metrics for regression and prob